### PR TITLE
feat(discover): Add new sort options

### PR DESCRIPTION
**Pull request by shruthilayaj**
Added 'Most Popular' and 'Recently Viewed' sort
options to Discover and Dashboards Manager.

### CODE REVIEW

**Review by edwardgou-sentry — APPROVED** [review]

File: static/app/views/dashboardsV2/view.tsx [hunk 1]
```diff
@@ -23,6 +24,13 @@ type Props = RouteComponentProps<{orgId: string; dashboardId: string}, {}> & {
 };
 
 function ViewEditDashboard(props: Props) {
+  useEffect(() => {
+    const {api, organization, params} = props;
+    if (params.dashboardId && params.dashboardId !== 'default-overview') {
+      updateDashboardVisit(api, organization.slug, params.dashboardId);
+    }
+  });
```
Comment (Zylphrex) [inline static/app/views/dashboardsV2/view.tsx:32]: You'll want to specify a dependency array here to avoid this hook being called on every render.

Comment (shruthilayaj) [reply static/app/views/dashboardsV2/view.tsx:32]: This PR needs to be updated, it's stale 1 sec

Comment (shruthilayaj) [reply static/app/views/dashboardsV2/view.tsx]: Updated @Zylphrex And the thing you mentioned was updated in the previous [PR](https://github.com/getsentry/sentry/blob/master/static/app/views/dashboardsV2/view.tsx#L35)

**Comment by Zylphrex** [comment]
@shruthilayaj Is the intent to go to GA with this immediately?

**Comment by shruthilayaj** [comment]
> @shruthilayaj Is the intent to go to GA with this immediately?

Yeah. I'm going to wait maybe till next week to merge this PR so that we have some meaningful data for most popular and last viewed sorts.

**Review by Zylphrex — APPROVED** [review]

**Review by dashed — APPROVED** [review]

**Comment by shruthilayaj** [comment]
Did a bad merge with master, re-opened a new PR [here](https://github.com/getsentry/sentry/pull/28413/files)

**Review by untitaker — COMMENTED** [review]
ship it!

**Pull request closed without merging**